allennlp/common/plugins.py

```diff
@@ -86,8 +86,9 @@ def import_plugins() -> None:
             # For default plugins we recursively import everything.
             import_module_and_submodules(module_name)
             logger.info("Plugin %s available", module_name)
-        except ModuleNotFoundError:
-            pass
+        except ModuleNotFoundError as e:
+            if e.name != module_name:
```

Comment:
This is to check if the failure is in a submodule?

Reply:
Yea